src/borg/testsuite/archiver.py

```diff
@@ -2553,10 +2553,7 @@ def test_recreate_recompress(self):
         assert sha256_before == sha256_after
 
     def test_recreate_timestamp(self):
-        if sys.version_info >= (3, 6):
-            local_timezone = datetime.now(timezone(timedelta(0))).astimezone().tzinfo
-        else:
-            local_timezone = datetime.now(timezone.utc).astimezone().tzinfo
+        local_timezone = datetime.now(timezone(timedelta(0))).astimezone().tzinfo
```

Comment:
so, this also works for py35?

Reply:
According to my tests and Travis, yes :)